### PR TITLE
Refine Managed Subscribe Sample

### PR DESCRIPTION
**Pull request by devkelley**
The is PR does the following:
- Adds a README explaining the managed_subscribe sample with sequence diagrams.
- Updates the consumer to allow for the frequency constraint to be set by the command line call.
- Allows for multiple consumers to be run.
- Consumer now shuts down more efficiently.
- Adds comments to managed subscribe proto file.

### CODE REVIEW

File: docs/design/diagrams/managed_subscribe_module_sequence.puml [hunk 1]
```diff
@@ -0,0 +1,65 @@
+@startuml Managed Subscribe Module
+title Managed Subscribe Module Sequence
+participant DTProvider
```
Comment (jorchiu) [inline docs/design/diagrams/managed_subscribe_module_sequence.puml:3]: Maybe replace `DTProvider` and `DTConsumer` with `Digital Twin Provider` and `Digital Twin Consumer`  respectively to be consistent with the other sequence diagrams in Ibeji.

File: docs/design/diagrams/managed_subscribe_module_sequence.puml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+participant Ibeji
+participant ManagedSubscribeModule
+participant DTConsumer
+participant Agemo
+participant Broker
+
+autonumber
+
+DTConsumer -> Ibeji: FindById (Temp)
```
Comment (jorchiu) [inline docs/design/diagrams/managed_subscribe_module_sequence.puml:12]: Just curious what do you mean by `FindById (Temp)` here?

File: samples/managed_subscribe/README.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
+## Managed Subscribe Sample
+
+### Introduction
+
+The managed subscribe sample shows how ibeji can extend its functionality with modules to give
```
Comment (jorchiu) [inline samples/managed_subscribe/README.md:5]: Please capitalize `Ibeji` and `MQTT`

File: docs/design/diagrams/managed_subscribe_module_sequence.puml (hunk 2 above)
Comment (devkelley) [reply docs/design/diagrams/managed_subscribe_module_sequence.puml:12]: Looks like it was a holdover from an old diagram I started from, changed to entity_id

File: docs/design/diagrams/managed_subscribe_interceptor_sequence.puml [hunk 4]
```diff
@@ -0,0 +1,38 @@
+@startuml Managed Subscribe Interceptor
+title Managed Subscribe Interceptor Sequence
+participant DTProvider
```
[the remark this answers is not in the document]
Comment (devkelley) [reply docs/design/diagrams/managed_subscribe_interceptor_sequence.puml]: updated

File: docs/design/diagrams/managed_subscribe_module_sequence.puml (hunk 2 above)
Comment (devkelley) [reply docs/design/diagrams/managed_subscribe_module_sequence.puml]: updated

File: samples/managed_subscribe/README.md (hunk 3 above)
Comment (devkelley) [reply samples/managed_subscribe/README.md]: updated

File: docs/design/diagrams/managed_subscribe_module_sequence.puml (hunk 1 above)
Comment (devkelley) [reply docs/design/diagrams/managed_subscribe_module_sequence.puml]: changed

**Review by jorchiu — APPROVED** [review]

**Pull request merged**